store/src/java/com/zimbra/cs/service/account/GetInfo.java

```diff
@@ -461,4 +475,26 @@ protected Element encodeChildAccount(Element parent, Account child, boolean isVi
     private void doDiscoverRights(Element eRights, Account account, Set<Right> rights) throws ServiceException {
         DiscoverRights.discoverRights(account, rights, eRights, false);
     }
+
+    private boolean checkIfPowerpasteInstalled() {
+        String libLocation = "/opt/zimbra/common/lib/pasteitcleaned";
+        String extLoc = "/opt/zimbra/lib/ext/powerpaste-ext/zm-powerpaste-extension.jar";
+        try {
+            File lib = new File(libLocation);
+            File ext = new File(extLoc);
+            if (lib.exists() && lib.isDirectory() && lib.canRead()) {
+                if (ext.exists() && ext.isFile() && ext.canRead()) {
+                    URL url = new URL("http://localhost:5000");
+                    if (url.openConnection() != null) {
```

Comment:
As per api doc, https://docs.oracle.com/javase/7/docs/api/java/net/URL.html#openConnection(),
```
A new instance of URLConnection is created every time when invoking the URLStreamHandler.openConnection(URL) method of the protocol handler for this URL.

It should be noted that a URLConnection instance does not establish the actual network connection on creation. This will happen only when calling URLConnection.connect().
```
I think connect method needs to be called on url connection to check whether the connection is available or not. https://docs.oracle.com/javase/7/docs/api/java/net/URLConnection.html#connect()

Reply:
Will fix this